tests/integration/conftest.py

```diff
@@ -46,6 +50,74 @@ def testing_api_key(environ: str) -> str:
     return os.environ["LABELBOX_TEST_API_KEY_STAGING"]
 
 
+def beta_endpoint(fn):
```

Comment:
Is this beta_endpoint code copied from elsewhere? Why duplicate?

Reply:
This is slightly different and used just for testing. We are exposing functionality we don't want customers to use but only for testing.

Reply:
Removed.